From 97822e3cbc63d3bd110bc3c654b3fbb4d4d1904c Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Mon, 3 Mar 2014 11:57:34 +0100
Subject: [PATCH] Wizards should look for templates in Template_internal, not
 Template_user

...as only the former reliably denotes the share/template tree where the
wizard templates are stored.  (Presumably the latter defaulted to the former as
long as it wasn't explicitly set in the past, but is now always explicitly set
since 838b77f5f3d6d8fd98891e99a23ff78a6a357cb2 "Resolves: rhbz#1065807 use xdg
~/Templates for default Template location.")

Change-Id: I7d1ac47d821c778fe342865465d0505a94b824bf
---
 wizards/com/sun/star/wizards/ui/WizardDialog.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wizards/com/sun/star/wizards/ui/WizardDialog.py b/wizards/com/sun/star/wizards/ui/WizardDialog.py
index 6cf9bc5..4e41fdf 100644
--- a/wizards/com/sun/star/wizards/ui/WizardDialog.py
+++ b/wizards/com/sun/star/wizards/ui/WizardDialog.py
@@ -126,7 +126,7 @@ class WizardDialog(UnoDialog2):
         xPropertySet = \
             self.xMSF.createInstance("com.sun.star.util.PathSettings")
         self.sTemplatePath = \
-            xPropertySet.getPropertyValue("Template_user")[0]
+            xPropertySet.getPropertyValue("Template_internal")[0]
         self.sUserTemplatePath = \
             xPropertySet.getPropertyValue("Template_writable")
             
-- 
1.8.5.3

